### PR TITLE
AS4 completed

### PR DESCRIPTION
**Pull request by TheIvano**

### CODE REVIEW

File: TheIvano-180294/Task06.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+Automatically generated by Colaboratory.
+Original file is located at
+    https://colab.research.google.com/drive/1hSv61ejXESrp34CUz1PqZCsoA43tzc8y
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline TheIvano-180294/Task06.py]: will not work in python. This is notebook magic

File: TheIvano-180294/Task07.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+"""Task07.ipynb
+Automatically generated by Colaboratory.
+Original file is located at
+    https://colab.research.google.com/drive/1tV5j-DRcpPtoJGoMj8v2DSqR_9wyXeiE
+**Task 07: Querying RDF(s)**
+"""
+
+!pip
+install
+rdflib
```
Comment (dgarijo) [inline TheIvano-180294/Task07.py]: these three lines won't even work in a notebook

File: TheIvano-180294/Task07.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+# SPARQL
+# TO DO
+q2 = prepareQuery('''
+  SELECT DISTINCT ?s
+  WHERE { 
+    {
+      ?s rdf:type ns:Person. 
+    }
+    UNION {
+      ?p rdfs:subClassOf ns:Person.
```
Comment (dgarijo) [inline TheIvano-180294/Task07.py]: missing all class hierarchy. Use chained properties with *

File: TheIvano-180294/Task07.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+     } 
+    UNION {
+        ?s2 rdfs:subClassOf ns:Person.
+        ?s rdf:type ?s2 .
+       ?s ?p ?o
+      }
+    }
+  ''',
+                  initNs={"rdf": RDF, "rdfs": RDFS, "ns": NameSpaceitor}
+                  )
```
Comment (dgarijo) [inline TheIvano-180294/Task07.py]: same as above.
Also missing exercises in rdflib

**Review by dgarijo — CHANGES_REQUESTED** [review]

File: TheIvano-180294/Task07.py (hunk 2 above)
Comment (TheIvano) [reply TheIvano-180294/Task07.py]: I pressed enter by accident smh. I'll fix it

File: TheIvano-180294/Task06.py (hunk 1 above)
Comment (TheIvano) [reply TheIvano-180294/Task06.py]: just removed it

File: TheIvano-180294/Task07.py (hunk 3 above)
Comment (TheIvano) [reply TheIvano-180294/Task07.py]: you mean like " UNION {
      ?p rdfs:subClassOf* ns:Person." ??

File: TheIvano-180294/Task07.py (hunk 4 above)
Comment (TheIvano) [reply TheIvano-180294/Task07.py]: just uploaded a new version.

Keep me posted if something's not fine

File: TheIvano-180294/Task07.py (hunk 3 above)
Comment (dgarijo) [reply TheIvano-180294/Task07.py]: yes

**Review by dgarijo — APPROVED** [review]

**Pull request merged**